main.go

```diff
@@ -62,11 +66,19 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
-
+	// TODO: Set this with credentials on the server side
+	// TODO: This needs to be set so we don't fail when instantiating client
```

Comment:
I feel like this is most important todo, the other ones we can live with

Reply:
Yeah, I was thinking tls and generate key/pair before creating the server, however that is done outside of this PR to manipulate the grpc server.  _(The GRPC service is a deployment outside of the controller-manager)_.  The client setting up however, I can pass it credentials to interact with this.  I can clean up the TODO but need more cycles to do the grpc server credentialing on the server side.

Reply:
Sounds good, we should separate this into it's own issue perhaps